src/components/tutorial/TutorialNav.astro

```diff
@@ -106,6 +106,10 @@ const isCurrentUnit = (unit: typeof units[number]) =>
 		font-weight: bold;
 	}
 
+	:global(a:focus unit-progress-icon) {
+		outline: 2px solid var(--theme-accent-secondary);
```

Comment:
I now see a _double_ outline in Firefox but if this is fixing a scenario where it’s completely missing in some browsers, NWTWWHB.

<img width="147" alt="image" src="https://user-images.githubusercontent.com/357379/225456978-3dfa7034-3c18-4261-96a1-324c9b9f8863.png">

Reply:
Nice catch! I'm not entirely sure why this happens, but the `<a/>` in Firefox inherits the default focus style for links, but not Chrome (and probably all the other Chromium based browsers). I believe it might be related to `role="tab"` changing how the element is interpreted? In any case, changing `outline-offset` to match the inherited outline fixes this.

Reply:
Smart fix — thanks @Yan-Thomas!